app/controllers/home_page_controller.rb

```diff
@@ -1,5 +1,6 @@
 class HomePageController < ApplicationController
   def index
+    @category = Category.all
```

Comment:
```suggestion
    @categories = Category.all.pluck :id, :name
```
chỉ get ra các file cần dùng và categories phải số nhìu nhỉ, `,all` mà